### PR TITLE
Added .jpeg and .webp to be included by default

### PR DESCRIPTION
**Pull request by bjon**
## Rollup Plugin Name: `url`

This PR contains:

- [ ] bugfix
- [ ] feature
- [ ] refactor
- [ ] documentation
- [x] other

Are tests included?

- [x] yes (_bugfixes and features will not be merged without tests_)
- [ ] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

### Description

This change will include jpeg and webp files, to the `defaultInclude` list.

Also see: https://github.com/sveltejs/sapper-template/pull/285

### CODE REVIEW

File: packages/url/README.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ A [minimatch pattern](https://github.com/isaacs/minimatch), or array of patterns
 ### `include`
 
 Type: `String` | `Array[...String]`<br>
-Default: `['**/*.svg', '**/*.png', '**/*.jpg', '**/*.gif']`
+Default: `['**/*.svg', '**/*.png', '**/*.jp(e)?g', '**/*.gif', '**/*.webp']`
```
Comment (NotWoods) [inline packages/url/README.md:65]: Does `jpe?g` work?

**Review by NotWoods — CHANGES_REQUESTED** [review]
Looks good! Can you please add a test for the new extensions? Once that's done this should be good to merge.

File: packages/url/README.md (hunk 1 above)
Comment (bjon) [reply packages/url/README.md:65]: without () ?

Comment (bjon) [reply packages/url/README.md:65]: > Looks good! Can you please add a test for the new extensions? Once that's done this should be good to merge.

I'll try my best

**Review by NotWoods — APPROVED** [review]

**Comment by shellscape** [comment]
_Note for the future: We'll probably want to start using a thorough list of supported asset types, as @lukastaegert mentioned in another PR, should we get additional requests to expand this list._

thanks!

**Pull request merged**